dash/webapp/pages/toggles.js

```diff
@@ -258,44 +194,42 @@ const IndexPage = () => {
                 <p>{toggle.description}</p>
                 <Button
                   onClick={() => {
-                    setCookie(toggle.id, 'true', domain);
-                    // $FlowFixMe
-                    setToggles({
-                      ...toggles,
+                    setCookie(toggle.id, 'true');
+                    setToggleStates({
+                      ...toggleStates,
                       [toggle.id]: true,
                     });
                   }}
                   style={{
-                    opacity: toggles[toggle.id] === true ? 1 : 0.5,
+                    opacity: toggleStates[toggle.id] === true ? 1 : 0.5,
                   }}
                 >
                   👍 Count me in
                 </Button>
                 <Button
                   onClick={() => {
-                    setCookie(toggle.id, 'false', domain);
-                    // $FlowFixMe
-                    setToggles({
-                      ...toggles,
+                    setCookie(toggle.id, 'false');
+                    setToggleStates({
+                      ...toggleStates,
                       [toggle.id]: false,
                     });
                   }}
                   style={{
-                    opacity: toggles[toggle.id] === false ? 1 : 0.5,
+                    opacity: toggleStates[toggle.id] === false ? 1 : 0.5,
                   }}
                 >
                   👎 No thanks
                 </Button>
                 <Button
                   onClick={() => {
-                    setCookie(toggle.id, null, domain);
-                    // $FlowFixMe
-                    setToggles({
-                      ...toggles,
+                    setCookie(toggle.id, null);
+                    setToggleStates({
+                      ...toggleStates,
                       [toggle.id]: undefined,
                     });
                   }}
                   opaque
+                  States
```

Comment:
eh? (also, what's `opaque` doing?)

Reply:
Not sure where `States` came from though?